src/Features/CSharp/Portable/CodeRefactorings/CSharpRefactoringHelpersService.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System.Composition;
+using Microsoft.CodeAnalysis.CodeRefactorings;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.ExtractMethod;
+using Microsoft.CodeAnalysis.Host.Mef;
+
+namespace Microsoft.CodeAnalysis.CSharp.CodeRefactorings
+{
+    [ExportLanguageService(typeof(ISyntaxTriviaService), LanguageNames.CSharp), Shared]
+    class CSharpRefactoringHelpersService : RefactoringHelpersService
```

Comment:
Do you plan to add VisualBasic specific implementation in a separate PR?

Reply:
I completely forgot about that, will add it here.

Reply:
The implementation of that should be written in WB, right?

Reply:
Yes, it will in the VB Features project, which is in VB :)